lib/utils/date_models.dart

```diff
@@ -32,3 +32,24 @@ class Week {
     return 'Week{firstDay: $firstDay, lastDay: $lastDay}';
   }
 }
+
+enum WeekDay { monday, tuesday, wednesday, thursday, friday, saturday, sunday }
```

Comment:
why do you need an enum if dart provides named weekday variables, having a 
```dart
final weekDaysToHide = [Datetime.saturday, DateTime.sunday];
```
working with an integer list with days from 0-6 makes this code a whole lot less obfuscated.